src/Command/ImportDefaultDataCommand.php

```diff
@@ -0,0 +1,178 @@
+        protected AlertChangeTypeRepository $alertChangeTypeRepository,
+        protected ApplicationConfigRepository $applicationConfigRepository,
+        protected AssessmentOptionRepository $assessmentOptionRepository,
+        protected CompetencyRepository $competencyRepository,
+        protected CourseClerkshipTypeRepository $courseClerkshipTypeRepository,
+        protected CurriculumInventoryInstitutionRepository $curriculumInventoryInstitutionRepository,
+        protected LearningMaterialStatusRepository $learningMaterialStatusRepository,
+        protected LearningMaterialUserRoleRepository $learningMaterialUserRoleRepository,
+        protected SchoolRepository $schoolRepository,
+        protected SessionTypeRepository $sessionTypeRepository,
+        protected TermRepository $termRepository,
+        protected UserRoleRepository $userRoleRepository,
+        protected VocabularyRepository $vocabularyRepository
+    ) {
+        parent::__construct();
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function configure(): void
+    {
+        $this
+            ->setName('ilios:import-default-data')
+            ->setAliases(['ilios:setup:import-default-data'])
+            ->setDescription('Imports default application data into Ilios. Only works with an empty database schema.');
+    }
+
+    /**
+     * @inheritdoc
+     */
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        if (!$this->lock()) {
+            $io->error('The command is already running in another process.');
+            return Command::FAILURE;
+        }
+
+        $school = $this->schoolRepository->findDTOBy([]);
+        if ($school) {
+            $io->error('Your database already contains data. Aborting import process.');
+            return Command::FAILURE;
+        }
+
+        $io->info('Started data import, this may take a while...');
+        $referenceMap = [];
+        try {
+            // ACHTUNG!
+            // we MUST clear the the aamc_method and application_configs table as part of the import process,
+            // since it gets pre-populated with data in the previous step of the installation
+            // process (when migrations are running).
+            // So let's just clear all records out here first, in order to avoid data duplication issues.
+            // [ST 2021/07/28]
+            $this->aamcMethodRepository->deleteAll();
+            $this->applicationConfigRepository->deleteAll();
+
+            // now, let's import
+            $referenceMap = $this->dataLoader->import($this->aamcMethodRepository, 'aamc_method', $referenceMap);
+            $referenceMap = $this->dataLoader->import($this->aamcPcrsRepository, 'aamc_pcrs', $referenceMap);
+            $referenceMap = $this->dataLoader->import(
+                $this->aamcResourceTypeRepository,
+                'aamc_resource_type',
```

Comment:
I wonder if these types should be a constant on the entity or the repository. Otherwise they get passed around and used as strings which is OK, but a little un-clear where they come from.

Reply:
agreed. magic strings are iffy. will replace them with constants.

Reply:
:heavy_check_mark: